renderdoc/driver/vulkan/vk_debug.cpp

```diff
@@ -2052,12 +2052,15 @@ void VulkanDebugManager::FillWithDiscardPattern(VkCommandBuffer cmd, DiscardType
 
     uint32_t pass = 0;
 
+    VkImageSubresourceRange barrierDiscardRange = discardRange;
+    barrierDiscardRange.aspectMask = imAspects;
+
     VkImageMemoryBarrier dstimBarrier = {
         VK_STRUCTURE_TYPE_IMAGE_MEMORY_BARRIER, NULL,
         VK_ACCESS_ALL_READ_BITS | VK_ACCESS_ALL_WRITE_BITS, VK_ACCESS_COLOR_ATTACHMENT_WRITE_BIT,
```

Comment:
I notice here that dstAccessMask should probably be conditional either `VK_ACCESS_COLOR_ATTACHMENT_WRITE_BIT` or `VK_ACCESS_DEPTH_STENCIL_ATTACHMENT_WRITE_BIT`.

Reply:
good find :) (valid didn't catch it ^^)